modules/passkey/test/userstories/RotatePasskeyOwner.spec.ts

```diff
@@ -7,9 +7,9 @@ import {
   buildSafeUserOpTransaction,
   buildPackedUserOperationFromSafeUserOperation,
   signSafeOp,
-} from '@safe-global/safe-4337/src/utils/userOp'
-import { buildSignatureBytes } from '@safe-global/safe-4337/src/utils/execution'
-import { chainId } from '@safe-global/safe-4337/test/utils/encoding'
+} from '@safe-global/safe-4337/dist/src/utils/userOp'
+import { buildSignatureBytes } from '@safe-global/safe-4337/dist/src/utils/execution'
+import { chainId } from '@safe-global/safe-4337/dist/test/utils/encoding'
```

Comment:
I don't think this should be part of the user story code (I.e. we shouldn't depend on trivial test code).

Not related to this PR though.

Reply:
I can easily include it in this PR if you don't mind.

Reply:
Nope, don't mind as it is a very small change and somewhat related.